src/stretch/dynav/mapping_utils/voxel.py

```diff
@@ -395,6 +399,28 @@ def add(
             ]
             if len(selected_indices) == 0:
                 return
+
+            # Remove points that are too close to the robot
+            mesh = self.urdf_visualizer.get_combined_robot_mesh()
+
+            # Fetch bounds of the robot
+            bounds = mesh.bounds()
+
+            # Expand bounds by the local radius
+            bounds = bounds + np.array(
+                [
+                    [-0.1, -0.1],
```

Comment:
This is a very large step - 10cm - right?

Reply:
Correct. Trying out with a smaller value and testing with ai_pickup.